### PR TITLE
docs: move the md files to /docs/en/latest foleder

### PR DESCRIPTION
**Pull request by stu01509**
According to this apache/apisix-website#216, so I created the folder /docs/en/latest to save the current README.md file.

### CODE REVIEW

**Comment by tokers** [comment]
@juzhiyuan ju

File: docs/en/latest/config.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "version": 1.0,
+    "sidebar": [
+      {
+        "type": "category",
+        "label": "Installation",
+        "items": ["build", "manual", "example"]
+      }
+    ]
+  }
```
Comment (juzhiyuan) [inline docs/en/latest/config.json:10]: new line, please

File: docs/en/latest/config.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+    "version": 1.0,
+    "sidebar": [
+      {
+        "type": "category",
+        "label": "Installation",
+        "items": ["build", "manual", "example"]
```
Comment (juzhiyuan) [inline docs/en/latest/config.json:7]: Hi, those items should contain necessary meta information, please refer to `apisix`

**Comment by stu01509** [comment]
Hi @juzhiyuan 

Update the commit.

**Comment by juzhiyuan** [comment]
@stu01509 It seems that no changes happened.

File: docs/en/latest/config.json [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
\ No newline at end of file
+  "version": 1.0,
+  "sidebar": [
+    {
+      "type": "category",
+      "label": "Installation",
+      "items": [ "build", "manual", "example"]
+    }
+  ]
+}
```
Comment (juzhiyuan) [inline docs/en/latest/config.json]: new line

File: docs/en/latest/manual.md [hunk 4]
```diff
@@ -1,3 +1,27 @@
+---
+title: Manual deploy apisix via docker
```
Comment (juzhiyuan) [inline docs/en/latest/manual.md:2]:
```suggestion
title: Deploy Apache APISIX with Docker
```

File: docs/en/latest/build.md [hunk 5]
```diff
@@ -1,3 +1,26 @@
+---
+title: Build an image from source
```
Comment (juzhiyuan) [inline docs/en/latest/build.md:2]:
```suggestion
title: Build an image from the source codes
```

File: docs/en/latest/config.json (hunk 3 above)
Comment (stu01509) [reply docs/en/latest/config.json]: Sorry, the auto prettier remove this, update the commit :)

**Comment by stu01509** [comment]
Update the commit :)

**Review by juzhiyuan — APPROVED** [review]
LGTM

**Comment by juzhiyuan** [comment]
image

@stu01509 Hi, please fix the Lint issue when you are free :)

**Comment by stu01509** [comment]
> image
> 
> @stu01509 Hi, please fix the Lint issue when you are free :)

Update the commit

**Review by juzhiyuan — APPROVED** [review]

**Review by tokers — APPROVED** [review]

**Pull request merged**